packages/front-end/services/config.ts

```diff
@@ -31,6 +31,17 @@ export function useConfigJson({
       settings: {
         pastExperimentsMinLength: settings.pastExperimentsMinLength ?? 6,
         visualEditorEnabled: !!settings.visualEditorEnabled,
+        environments: settings.environments,
```

Comment:
I think I'm going to try and implement this like we're doing with metrics to make it a bit cleaner.